mhlib/mhConfig.go

```diff
@@ -27,6 +27,7 @@ type MHConfig struct {
 	Simulate      bool     `yaml:"simulate"`
 	TargetContext string   `yaml:"targetContext"`
 	Team          string   `yaml:"team"`
+	CLIValues     []string `yaml:"cliValues"`
```

Comment:
Please rename this to something more descriptive, `CLIValues` gives the impression of this field holding all CLI flags. E.g. `SETValues` like you did in `cmd/`.